py2pack/requires.py

```diff
@@ -26,9 +27,7 @@
 def _requirement_filter_by_marker(req):
     """check if the requirement is satisfied by the marker"""
     if hasattr(req, 'marker') and req.marker:
-        # TODO (toabctl): currently we hardcode python 2.7 and linux2
-        # see https://www.python.org/dev/peps/pep-0508/#environment-markers
-        marker_env = {'python_version': '2.7', 'sys_platform': 'linux'}
+        marker_env = {'python_version': '.'.join(map(str, sys.version_info[:2])), 'sys_platform': sys.platform}
```

Comment:
https://www.python.org/dev/peps/pep-0508/#environment-markers says:
> (note that "linux" is from Python3 and "linux2" from Python2) 

I seems this is more correct now than it was before even for py2.